Funcky.Test/TestUtils/PriorityQueueArbitrary.cs

```diff
@@ -0,0 +1,19 @@
+#if PRIORITY_QUEUE
+using FsCheck;
+
+namespace Funcky.Test.Extensions;
+
+public class PriorityQueueArbitrary
+{
+    public static Arbitrary<PriorityQueue<string, int>> GeneratePriorityQueue()
+        => GeneratePriorityQueueGenerator()
+            .ToArbitrary();
+
+    public static Gen<PriorityQueue<string, int>> GeneratePriorityQueueGenerator()
+        => from values in Arb.Generate<List<string>>()
+            from seed in Arb.Generate<int>()
+            let randomPriority = new System.Random(seed)
+            let priorityQueue = new PriorityQueue<string, int>(values.Select(v => (v, randomPriority.Next())))
```

Comment:
There's `Gen.SequenceToList` that turns an `IEnumerable<Gen<T>>` into a `Gen<IEnumerable<T>>`. I think that could help here.

If fscheck supports tuples, maybe you could even do `Arb.Generate<List<(string, int)>>()`?

Reply:
Fixed in cc757f1d0a5bba0afc92d5d0537d3c8e60f76711
* Generator written vor `ValueTuple'2`